### ISSUE TITLE
The ability to switch off real-time code editor linting in the settings

### ISSUE THREAD

**Issue by QuincyLarson** [freeCodeCamp/freeCodeCamp#16973]
CodeMirror has linting errors that pop up when you type invalid code:

![basic_javascript__comment_your_javascript_code___freecodecamp](https://user-images.githubusercontent.com/985197/38003338-c57d0590-31fc-11e8-9a55-abcf52a05868.png)

A camper suggested we allow for them to toggle these off. Is this a feature we can easily support, and if so, should we support it?

**Comment by raisedadead** [comment]
@QuincyLarson we have debated over this in the past. And I believe we should not.

Reasons in short:

1. Support requests:
"My code is correct, but its not working"! - Usual suspects, "You missed a comma there, or that brace there, etc."

2. Habit building as debated in https://github.com/freeCodeCamp/freeCodeCamp/issues/16509
Its easy to fall into a practice of OKAY practices for example as popularized by https://standardjs.com/#standardjs--the-rules

    That's a good thing, but the tradeoff for our audience is not justified IMHO, who are more often than just getting started to coding.

I would rather propose to have a lesson, following which you fall into a practice of typing code where you never see a warning.

Ex:
```
Step 1:
function ()

Step 2:
function () {}

Step 3:
function (paramA, ParamB) {
}

Step 4:
function (paramA, ParamB) {
 ...
}

```

And so on. I just typed that snippet out of muscle memory. I don't think I would be getting errors, in any steps.

Where this one would give me nightmares;

Ex:
```
Step 1:
function (

Step 2:
function (paramA, ParamB

Step 3:
function (paramA, ParamB) 

Step 4:
function (paramA, ParamB) {
 ...

```

So, its worth reinforcing this paradigm. 

Adding a setting will make things only more complex, and we will keep getting issues that aren't worth the time for the sake convenience these offer.

BUT, that said all I am taking about is our own editor. Users should of course do whatever they wish in their own setups, for projects, local editors etc.
